.github/workflows/tests.yml

```diff
@@ -20,19 +20,13 @@ jobs:
       - run: scripts-dev/config-lint.sh
 
   lint:
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        toxenv:
-          - "check_codestyle"
-          - "check_isort"
-          - "mypy"
-
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
-      - run: pip install tox
-      - run: tox -e ${{ matrix.toxenv }}
+    # This does a vanilla `poetry install` - no extras. I'm slightly anxious
+    # that we might skip some typechecks on code that uses extras. However,
+    # I think the right way to fix this is to mark any extras needed for
+    # typechecking as development dependencies. To detect this, we ought to
+    # turn up mypy's strictness: disallow unknown imports and be accept fewer
```

Comment:
Just curious if this has been done or is waiting on something?

Reply:
Not done, not waiting on anything in particular.

(We could easily pass extras on to the workflow---but I'm reluctant to, because I want the dev dependencies automatically pulled in by `poetry install` to be _exactly_ what you need to do all development tasks, including typechecking.)